docs/medical-api/api-reference/document/post-upload-url.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Post Upload URL"
```

Comment:
This title would confuse me if I was just reading the list of available endpoints. Perhaps `Upload Document` instead?